lib/active_admin/views/footer.rb

```diff
@@ -6,21 +6,17 @@ def build(namespace)
         super id: "footer"
         @namespace = namespace
 
-        if footer?
+        if footer_text.present?
           para footer_text
         else
           para powered_by_message
         end
       end
 
-      def footer?
-        @namespace.footer.present?
-      end
-
       private
 
       def footer_text
-        helpers.render_or_call_method_or_proc_on(self, @namespace.footer)
+        @namespace.footer(self)
```

Comment:
@varyonic maybe we should cache such stuff  using instance variable  since you now use 
```footer_text.present?```